examples/mount.rs

```diff
@@ -0,0 +1,22 @@
+#[macro_use] extern crate nickel;
+
+use nickel::{Nickel, Mount, StaticFilesHandler};
+
+fn main() {
+    let mut server = Nickel::new();
+
+    server.utilize(Mount::new("/a/b/c", middleware! {
+        "Hello world."
+    }));
+
+    server.utilize(Mount::new("/1/2/3", middleware! {
+        "H3110 w0r1d."
```

Comment:
I'm not sure these first two usages show anything useful to readers, perhaps printing `req.path_without_query()` would make things clearer?